shaders/src/getters.fs

```diff
@@ -65,6 +65,14 @@ float getNdotV() {
     return shading_NoV;
 }
 
+mat3 getTriplanarOrientationMatrix() {
+    return objectUniforms.triplanarOrientationMatrix;
+}
+
+vec3 getObjectAabbCenterWorldSpace() {
+    return objectUniforms.worldAabbCenter;
+}
```

Comment:
Please document in `Materials.md.html`, helps to navigate coordinate systems.

Reply:
Fixed in a1c7b04